src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -155,6 +152,50 @@ def test_create_in_different_group(self, resource_group, resource_group_location
         ])
 
 
+# Test Framework is not able to handle binary file format, hence, only run live
+class AppServiceLogTest(LiveScenarioTest):
+    @ResourceGroupPreparer()
+    def test_download_win_web_log(self, resource_group):
+        import zipfile
+        from pathlib import Path
+        webapp_name = self.create_random_name(prefix='webapp-win-log', length=24)
+        plan = self.create_random_name(prefix='win-log', length=24)
+        self.cmd('appservice plan create -g {} -n {}'.format(resource_group, plan))
+        self.cmd('webapp create -g {} -n {} --plan {} --deployment-source-url https://github.com/yugangw-msft/azure-site-test.git -r "node|6.1"'.format(resource_group, webapp_name, plan))
```

Comment:
Is there another standardized source of sample project?

Reply:
So far, no. I can move the sample test project to an azure official repository, but figured better not
1. The sample is very simple and pretty much was copied and pasted from Node Express web sample. If lost, which hasn't happened, it is fairly trivial to recover.
2. It is tailor'd for CLI specific test scenarios, say outputting the node/python stack version, which is not very suitable to be used for general samples.

Reply:
Ok. Filed issue https://github.com/Azure/azure-cli/issues/4680 to tracking.